service/build.gradle

```diff
@@ -55,23 +55,23 @@ sourceSets {
     resources.srcDir file('src/integrationTest/resources')
   }
 
-    smokeTest {
-        java {
-            compileClasspath += main.output
-            runtimeClasspath += main.output
-            srcDir file('src/smokeTest/java')
-        }
-        resources.srcDir file('src/smokeTest/resources')
-    }
+  smokeTest {
```

Comment:
unnecessary changes

Reply:
This is just the automatic tidying up in IntelliJ.

Reply:
Understandable but if you revert then it is less to review and doesn't unnecessarily mess with the git history

Reply:
Ok. Will do.